project/plugins.sbt

```diff
@@ -1,3 +1,6 @@
-addSbtPlugin("com.codecommit" % "sbt-spiewak-sonatype" % "0.21.0")
+addSbtPlugin("com.codecommit" % "sbt-spiewak-sonatype" % "0.23.0")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
-addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.2.24")
+addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.3.6")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.11.0")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.21.0")
```

Comment:
Hopefully it's now possible to remove the bundler plugin (and the lockfile)? Recently it does not seem to be very well maintained.

Reply:
I have tried to remove, but if I do, munit complains with this error:
```
error] munit.internal.JSFs$ needs to be imported from module 'fs' but module support is disabled
[error]   called from private java.nio.file.Files$.$anonfun$readAllBytes$1(java.nio.file.Path)[byte
[error]   called from java.nio.file.Files$.readAllBytes(java.nio.file.Path)[byte
[error]   called from java.nio.file.Files$.readAllLines(java.nio.file.Path)java.util.List
[error]   called from private munit.internal.console.Lines.$anonfun$formatLine$1(java.nio.file.Path)[java.lang.String
[error]   called from munit.internal.console.Lines.formatLine(munit.Location,java.lang.String,munit.Clues)java.lang.String
...
```

Reply:
Upgrading munit solved the issue